samples/nrf9160/http_update/application_update/README.rst

```diff
@@ -94,7 +94,7 @@ Remember to do the following when specifying the filenames:
 * Use the ``<bucket-name>.s3.<region>.amazonaws.com`` part of the URL as the hostname of the server hosting the images, without including ``https://``.
 * Specify the file names as the remaining part of the URL.
 
-You can then use the steps mentioned in the `Specifying the image files`_ section above to upload both images.
+Use the steps mentioned in the `Specifying the image files`_ section to upload both images.
```

Comment:
Not sure about using steps. Maybe follow the steps?

Reply:
Yes, that sounds better. Thanks!